app/views/acceptances/new.html.erb

```diff
@@ -1,6 +1,13 @@
-<%= simple_form_for @invitation, :url => invitation_acceptances_path(@invitation), :method => :post do |form| %>
-  <%= form.input :first_name, :required => true %>
-  <%= form.input :last_name, :required => true %>
-  <%= form.input :password, :required => true %>
-  <%= submit_tag "Accept the invitation!" %>
-<% end %>
+<div class="flexbox vertical center">
+  <section class="invitation-form">
+    <h1>Accept the trustee invitation</h1>
```

Comment:
We shouldn't have any actual strings of text in the views - this string should be moved to the `config/locales/en.yml` file under the `acceptances/new/title` key and then we can get it translated into other languages in the future. Also when it's moved let's make this "Accept your trustee invitation"

Reply:
This code was added because this form remained unstyled during the clearance upgrade. It adopts the same pattern found in `app/views/invitations/new.html.erb`. That form also has the actual string of text in the view. So, both ought to be fixed then.

Reply:
OK, we'll take care of this in another commit.